cmd/ocm/create/cluster/cmd.go

```diff
@@ -36,18 +38,23 @@ import (
 )
 
 var args struct {
-	dryRun bool
-
-	region            string
-	version           string
-	flavour           string
-	provider          string
-	expirationTime    string
-	expirationSeconds time.Duration
-	private           bool
-	multiAZ           bool
-	ccs               c.CCS
-	gcpServiceAccount arguments.FilePath
+	// positional args
+	clusterName string
+
+	// flags
+	interactive bool
+	dryRun      bool
+
+	region                string
+	version               string
+	flavour               string
+	provider              string
+	expirationTime        string
+	expirationSeconds     time.Duration
+	private               bool
+	multiAZ               bool
+	ccs                   c.CCS
```

Comment:
@cben Not related to these changes  - I see that 	`github.com/openshift-online/ocm-cli/pkg/cluster` is imported twice:
```
	"github.com/openshift-online/ocm-cli/pkg/cluster"
	c "github.com/openshift-online/ocm-cli/pkg/cluster"
```

Can you please fix that?

Reply:
#203